### PR TITLE
[TVEA-2291] Fixed Warnings

### PR DESCRIPTION
**Pull request by JackFrisoli**

### CODE REVIEW

**Review by psvnanda — APPROVED** [review]

File: NVActivityIndicatorView/Animations/NVActivityIndicatorAnimationSquareSpin.swift [hunk 1]
```diff
@@ -21,9 +21,9 @@ class NVActivityIndicatorAnimationSquareSpin: NVActivityIndicatorAnimationDelega
         animation.timingFunctions = [timingFunction, timingFunction, timingFunction, timingFunction]
         animation.values = [
             NSValue(caTransform3D: CATransform3DConcat(createRotateXTransform(angle: 0), createRotateYTransform(angle: 0))),
-            NSValue(caTransform3D: CATransform3DConcat(createRotateXTransform(angle: CGFloat(M_PI)), createRotateYTransform(angle: 0))),
-            NSValue(caTransform3D: CATransform3DConcat(createRotateXTransform(angle: CGFloat(M_PI)), createRotateYTransform(angle: CGFloat(M_PI)))),
-            NSValue(caTransform3D: CATransform3DConcat(createRotateXTransform(angle: 0), createRotateYTransform(angle: CGFloat(M_PI)))),
+            NSValue(caTransform3D: CATransform3DConcat(createRotateXTransform(angle: CGFloat(Double.pi)), createRotateYTransform(angle: 0))),
```
Comment (JALsnipe) [inline NVActivityIndicatorView/Animations/NVActivityIndicatorAnimationSquareSpin.swift]: Why not just use `CGFloat.pi`?

**Review by JALsnipe — CHANGES_REQUESTED** [review]
I feel like there are multiple places where we could be using `CGFloat.pi` but aren't.  Let's re-review this to make sure we're not initializing CGFloats from Doubles for no reason.

File: NVActivityIndicatorView/Animations/NVActivityIndicatorAnimationSquareSpin.swift (hunk 1 above)
Comment (JackFrisoli) [reply NVActivityIndicatorView/Animations/NVActivityIndicatorAnimationSquareSpin.swift]: I made the changes using replace all, the old pi constants were all doubles and I didn't realize that in Swift 3 there were now non-double constants for pi available as well and that it meant I could optimize some casting that wouldn't have been possible before. So, no good reason, just a bit of ignorance.

I'll push a commit with the redundant casting stripped out.

Comment (JALsnipe) [reply NVActivityIndicatorView/Animations/NVActivityIndicatorAnimationSquareSpin.swift]: No worries, thanks.

**Comment by JackFrisoli** [comment]
This is ready for review again.

**Review by JALsnipe — APPROVED** [review]

**Pull request merged**